docker-deploy/README.md

```diff
@@ -207,21 +207,6 @@ If the test passed, the output may look like the following:
 docker exec -it confs-10000_client_1 bash
 ```
 
-##### Modifying examples/upload_host.json
-
-```bash
-cat > fateflow/examples/upload/upload_host.json <<EOF
-{
-  "file": "examples/data/breast_hetero_host.csv",
-  "id_delimiter": ",",
-  "head": 1,
-  "partition": 10,
-  "namespace": "experiment",
-  "table_name": "breast_hetero_host"
-}
-EOF
-```
-
```

Comment:
Why remove this？

Reply:
This file is already there with the exactly the same content.